Doc/library/dis.rst

```diff
@@ -567,6 +567,17 @@ the original TOS1.
     .. versionchanged:: 3.11
        Exception representation on the stack now consist of one, not three, items.
 
+
+.. opcode:: END_THROW
```

Comment:
`END` suggests that this is the end of a sequence of bytecodes.
Maybe `CLEANUP_THROW` as it cleans up `_gen_throw`?